### PR TITLE
Re-enable readable-stream

### PR DESCRIPTION
**Pull request by mcollina**
#1037 

##### Checklist

- [ ] `npm test` passes
- [ ] tests are included
- [ ] documentation is changed or added
- [ ] contribution guidelines followed
      [here](https://github.com/nodejs/citgm/blob/HEAD/CONTRIBUTING.md)

### CODE REVIEW

**Comment by mcollina** [comment]
GH Action run: https://github.com/nodejs/citgm/actions/runs/7219998821

**Comment by mcollina** [comment]
CITGM: https://ci.nodejs.org/view/Node.js-citgm/job/citgm-smoker/3360/

**Comment by targos** [comment]
Looks broken on v21.

**Comment by mcollina** [comment]
working on it in https://github.com/nodejs/readable-stream/pull/528, hopefully landing soon.

**Comment by richardlau** [comment]
Installation fails on AIX as `esbuild` doesn't support it.

https://ci.nodejs.org/view/Node.js-citgm/job/citgm-smoker/3360/nodes=aix72-ppc64/console

<details>

```console
09:34:22 warn: readable-stream npm-install:| npm ERR! code 1                                                                                                                                                        
09:34:22 warn:                             | npm ERR! path /home/iojs/tmp/citgm_tmp/cd2c8a76-4c62-439c-840f-095946746320/readable-stream/node_modules/esbuild                                                       
09:34:22 warn:                             | npm ERR! command failed                                                                                                                                                
09:34:22 warn:                             | npm ERR! command sh -c node install.js                                                                                                                                 
09:34:22 warn:                             | npm ERR! /home/iojs/tmp/citgm_tmp/cd2c8a76-4c62-439c-840f-095946746320/readable-stream/node_modules/esbuild/install.js:69                                              
09:34:22 warn:                             | npm ERR!     throw new Error(`Unsupported platform: ${platformKey}`);                                                                                                  
09:34:22 warn:                             | npm ERR!           ^                                                                                                                                                   
09:34:22 warn:                             | npm ERR!                                                                                                                                                               
09:34:22 warn:                             | npm ERR! Error: Unsupported platform: aix ppc64 BE                                                                                                                     
09:34:22 warn:                             | npm ERR!     at pkgAndSubpathForCurrentPlatform (/home/iojs/tmp/citgm_tmp/cd2c8a76-4c62-439c-840f-095946746320/readable-stream/node_modules/esbuild/install.js:69:11)  
09:34:22 warn:                             | npm ERR!     at checkAndPreparePackage (/home/iojs/tmp/citgm_tmp/cd2c8a76-4c62-439c-840f-095946746320/readable-stream/node_modules/esbuild/install.js:211:28)          
09:34:22 warn:                             | npm ERR!     at Object.<anonymous> (/home/iojs/tmp/citgm_tmp/cd2c8a76-4c62-439c-840f-095946746320/readable-stream/node_modules/esbuild/install.js:238:1)               
09:34:22 warn:                             | npm ERR!     at Module._compile (node:internal/modules/cjs/loader:1375:14)                                                                                             
09:34:22 warn:                             | npm ERR!     at Module._extensions..js (node:internal/modules/cjs/loader:1434:10)                                                                                      
09:34:22 warn:                             | npm ERR!     at Module.load (node:internal/modules/cjs/loader:1206:32)                                                                                                 
09:34:22 warn:                             | npm ERR!     at Module._load (node:internal/modules/cjs/loader:1022:12)                                                                                                
09:34:22 warn:                             | npm ERR!     at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:142:12)                                                                    
09:34:22 warn:                             | npm ERR!     at node:internal/main/run_main_module:28:49                                                                                                               
09:34:22 warn:                             | npm ERR!                                                                                                                                                               
09:34:22 warn:                             | npm ERR! Node.js v22.0.0-pre                                                                                                                                           
09:34:22 warn:                             |                                                                                                                                                                        
09:34:22 warn:                             | npm ERR! A complete log of this run can be found in: /home/iojs/tmp/citgm_tmp/cd2c8a76-4c62-439c-840f-095946746320/home/.npm/_logs/2023-12-15T09_44_41_045Z-debug-0.log
```
</details>

**Comment by merceyz** [comment]
@richardlau I opened an issue in the esbuild repo for adding `aix ppc64 BE` support, do you think it be possible to test if esbuild works on that machine?

Ref https://github.com/evanw/esbuild/issues/3549#issuecomment-1858935215

**Comment by mcollina** [comment]
CITGM: https://ci.nodejs.org/view/Node.js-citgm/job/citgm-smoker/3363/

**Comment by mcollina** [comment]
GHA: https://github.com/nodejs/citgm/actions/runs/7219998821

**Comment by mcollina** [comment]
It seems it's passing on Linux. Why is it run on aix and windows even if it's skipped in the definition? https://ci.nodejs.org/view/Node.js-citgm/job/citgm-smoker/3363/nodes=aix72-ppc64/testReport/junit/(root)/citgm/readable_stream_v4_5_0/

**Comment by targos** [comment]
Skips only apply to the `citgm-all` command.

File: lib/lookup.json [hunk 1]
```diff
@@ -345,6 +345,11 @@
     "maintainers": "kriskowal",
     "skip": ["win32", ">=21"]
   },
+  "readable-stream": {
+    "prefix": "v",
+    "maintainers": "nodejs/streams",
+    "skip": ["win32", "ppc", "aix"]
```
Comment (richardlau) [inline lib/lookup.json:351]:
```suggestion
    "skip": ["win32", "aix"]
```

[Passes on Linux on ppc64le](https://ci.nodejs.org/view/Node.js-citgm/job/citgm-smoker/3363/nodes=rhel8-ppc64le/testReport/(root)/citgm/readable_stream_v4_5_0/), so no need to skip for "ppc".

**Comment by mcollina** [comment]
@RafaelGSS @richardlau what's the next step here?

**Comment by RafaelGSS** [comment]
> @RafaelGSS @richardlau what's the next step here?

If that's passing in the expected environments we can merge it and release a new version.

**Review by RafaelGSS — APPROVED** [review]

**Review by richardlau — APPROVED** [review]

**Comment by richardlau** [comment]
> @richardlau I opened an issue in the esbuild repo about adding `aix ppc64 BE` support, do you think it would be possible to test if it works on that machine?
> 
> Ref [evanw/esbuild#3549 (comment)](https://github.com/evanw/esbuild/issues/3549#issuecomment-1858935215)

@merceyz I've asked @abmusse from IBM to take a look.

**Comment by mcollina** [comment]
Note that the `esbuild` package is not strictly needed to run those tests, but I'm not sure how I can skip its install at this stage.

**Comment by mcollina** [comment]
CITGM: https://ci.nodejs.org/view/Node.js-citgm/job/citgm-smoker/3372/

**Comment by mcollina** [comment]
Should be good to go if the run pass. What are the next steps?

**Review by targos — APPROVED** [review]

**Pull request merged**